src/cplscheme/impl/Extrapolation.cpp

```diff
@@ -1,70 +1,73 @@
-#include "time/Waveform.hpp"
+#include "Extrapolation.hpp"
 #include <algorithm>
+#include "cplscheme/CouplingScheme.hpp"
 #include "logging/LogMacros.hpp"
 #include "utils/EigenHelperFunctions.hpp"
 
 namespace precice {
-namespace time {
+namespace cplscheme {
```

Comment:
IIRC we normally also use an `impl` namespace here.

Reply:
Yes. I will add this namespace.